tests/unit/amo/helpers.js

```diff
@@ -353,15 +353,25 @@ export const createFakeCollectionDetail = ({
   };
 };
 
-export const createFakeCollectionAddons = ({ addons = [fakeAddon] } = {}) => {
+export function createFakeCollectionAddon({
+  addon = fakeAddon, notes = null,
+} = {}) {
+  return { addon, notes };
+}
+
+export const createFakeCollectionAddons = ({
+  addons = [createFakeCollectionAddon()],
+} = {}) => {
   return addons.map((addon) => ({
-    addon,
+    addon: addon.addon,
     downloads: 0,
-    notes: null,
+    notes: addon.notes,
```

Comment:
You could also destructure the unique `map` parameter: `.map({ addon, notes }) => ({`